### PR TITLE
PUB-1673 SJP Public List validation schema update

### PR DESCRIPTION
**Pull request by KianKwa**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/PUB-1673

### Change description ###

SJP Public List validation schema update

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/resources/schemas/single_justice_procedure_public.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
           "examples": [
             "PR1 2LL"
```
Comment (ChrisS1512) [inline src/main/resources/schemas/single_justice_procedure_public.json:12]: Can we update the example here too to reflect the change

File: src/main/resources/schemas/single_justice_procedure_public.json [hunk 2]
```diff
@@ -159,7 +159,7 @@
                                                     "John Smith"
```
Comment (ChrisS1512) [inline src/main/resources/schemas/single_justice_procedure_public.json:159]: Example to be updated here

**Review by ChrisS1512 — CHANGES_REQUESTED** [review]

**Review by junaidiqbalmoj — APPROVED** [review]

File: src/main/resources/schemas/single_justice_procedure_public.json (hunk 1 above)
Comment (KianKwa) [reply src/main/resources/schemas/single_justice_procedure_public.json]: Example updated

File: src/main/resources/schemas/single_justice_procedure_public.json (hunk 2 above)
Comment (KianKwa) [reply src/main/resources/schemas/single_justice_procedure_public.json]: Example updated

**Review by ChrisS1512 — APPROVED** [review]

**Pull request merged**